### DISCUSSION TITLE
Formily 1.x，SchemaForm传递props会丢失对象原型

### DISCUSSION THREAD

**Discussion by ironicstone** [alibaba/formily#1833] [Q&A 1.x] [answered]
如题，通过SchemaForm的x-component-props传递数据，会丢失对象的原型。

复现case: https://codesandbox.io/s/exciting-night-q0utz?file=/index.js

**Comment by janryWang** [comment, marked as answer]
用context传递，1.x会对props做脏检查，所以做了深拷贝
